### PR TITLE
[SINT-5479] Integrated tox and GitHub Workflows to run unit tests with CI/CD

### PR DESCRIPTION
**Pull request by marcos-vivar**
## TL;DR

- Replicated tox configuration from **[TS SDK v2](https://github.com/trustar/trustar-sdk2-proto/blob/main/tox.ini)** for for TS SDK v1
- Replicated GitHub's Workflows configuration from **[TS SDK v2](https://github.com/trustar/trustar-sdk2-proto/blob/main)** for TS SDK v1

## What?
**TS (Legacy) SDK v1** does not support yet **tox** nor has a CI/CD integration with this repository to run the test suite (as a standard to run tests within sandboxed environments), and can be safely (currently attempting to be) integrated as already being done on [**TS (Legacy) SDK v2**](https://github.com/trustar/trustar-sdk2-proto/tree/main/.github/workflows).

### CODE REVIEW

**Comment by marcos-vivar** [comment]
Hi @dsolmirano:
The pipeline for the CI/CD configuration run successfully ✅

🚧 
However, tests failed and that will be addressed in **Python 3.9 upgrade**:
https://github.com/trustar/trustar-python/pull/112

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,21 @@
+
+[tox]
+envlist = py38
```
Comment (dsolmirano) [inline tox.ini:3]: Does this mean we can only move to 3.8? or is it possible to move to 3.9?

**Review by dsolmirano — APPROVED** [review]
LGTM however I have a question regarding the version on tox

File: tox.ini (hunk 1 above)
Comment (marcos-vivar) [reply tox.ini:3]: Hey @dsolmirano 👋🏽 

The current update to **Python3.9** and everything related to it will be done on https://github.com/trustar/trustar-python/pull/112

File: tox.ini [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    pre-commit
+    pytest
+    pytest-cov
+    pytest-mock
+    pytest-xdist
+    requests-mock
+    twine
+
+commands =
+    python3 --version
```
Comment (marcos-vivar) [inline tox.ini:21]: @dsolmirano and @nicomitchell 👋🏽 
Added a simple Python command to make the pipeline go ✅ and left a comment that Unit Tests are being addressed on https://github.com/trustar/trustar-python/pull/112
👍🏽 
Thanks for the feedback!

**Review by dsolmirano — APPROVED** [review]

**Review by nicomitchell — APPROVED** [review]
